doc/ping.xml

```diff
@@ -118,6 +118,46 @@ xml:id="man.ping">
     IPv6 source routing was deprecated (RFC5095).</para>
   </refsection>
 
+  <refsection>
+    <info>
+      <title>IPV6 LINK-LOCAL DESTINATIONS</title>
+    </info>
+    <para>For IPv6, when the destination address has link-local scope and
+    ping is using <emphasis remap="I">ICMP datagram sockets</emphasis>,
+    the output interface must be specified.
+    When <command>ping</command> is using raw sockets, it is not strictly
```

Comment:
For consitency, should "raw" also have emphasis?

Reply:
yep